test/nodes/test_agent_rocketride.py

```
@@ -0,0 +1,404 @@
+        inst = IInstance()
+        inst.IGlobal = MagicMock()
+        inst.instance = MagicMock()
+
+        # Non-tool op should go to super().invoke()
+        param = {'op': 'lifecycle.close'}
+        # This may raise or return depending on base impl; we just verify
+        # agent.handle_invoke is NOT called
+        try:
+            inst.invoke(param)
+        except Exception:
+            pass  # Base class may not handle this
```

Ruff (0.15.7)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)